src/Runtime/Optimized/Physics/mmdBulletPhysics.ts

```
@@ -0,0 +1,640 @@
+import { DeepImmutable, Nullable } from "@babylonjs/core/types";
```

ESLint

[error] All imports in the declaration are only used as types. Use `import type`.

(@typescript-eslint/consistent-type-imports)